test/test_helper.rb

```diff
@@ -1,7 +1,9 @@
+# typed: false
 # frozen_string_literal: true
 
 $LOAD_PATH.unshift(File.expand_path("../lib", __dir__))
 
+require "sorbet-runtime"
 require "ruby-lsp"
 require "syntax_tree"
 require "ruby_lsp/cli"
```

Comment:
We wanted to not require anything on the default require, to avoid polluting loaded features. I created an internal file like we do in Tapioca do group all necessary requires.